examples/xlnet/xlnet_generation_main.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Example of building XLNet language model for sample generation.
+"""
+
+import argparse
+import torch
+import sentencepiece as spm
+
+import xlnet
+import xlnet.model.decoder
+from texar.modules.decoders.decoder_helpers import TopKSampleEmbeddingHelper, \
+    TopPSampleEmbeddingHelper
```

Comment:
These should be imported from `texar.modules`. The previous discussion was to not expose too much internal details to the user.

Reply:
Done in 0e23eda